bin/gitmeta.jl

```diff
@@ -3,7 +3,7 @@
 import Pkg.TOML
 import LibGit2
 
-const STDLIBS = [
+const STDLIBS = sort!(by = lowercase, [
```

Comment:
TIL you can use kwargs like this, thats pretty neat.

Reply:
Yup, keyword and positional args can be interleaved, a nice aspect of keeping them distinct.